exporter/containerimage/export.go

```diff
@@ -115,6 +115,10 @@ func (e *imageExporterInstance) Export(ctx context.Context, src exporter.Source)
 		e.opt.ImageWriter.ContentStore().Delete(context.TODO(), desc.Digest)
 	}()
 
+	if n, ok := src.Metadata["image.name"]; e.targetName == "" && ok {
```

Comment:
I think the user should say that they allow frontend to set/tweak the image name with the request. It shouldn't be unexpected. If the user says `name=%s` (or the template solution) it is clear that they expect part of it to be replaced. Also, make sure to return the final image name with solve metadata.

Reply:
I thought this would do `fmt.Sprintf` then so `%s` can be anywhere in the string. If not and it is only a constant then maybe `%` or `*` is less confusing as a placeholder. For the templating, should we do it, it is probably better to use the builtin one for consistency.

Reply:
Perhaps (very likely) this isn't such a big concern in Go as it is with e.g. C, but passing a user controlled format string to a `printf` type function scares me. In that case perhaps `%` or `*` is better (I think I prefer the latter).

I was considering using `${image.name}` (i.e. that exact literal string) such that it remains valid if/when we add proper templating in the future.

I didn't find an existing/builtin one (unless you meant `text/template`?). I'll have another look though.

Reply:
Ok, can we switch to `*` then

Reply:
> Ok, can we switch to `*` then

Ack. Just pushed that change.